pkg/ttl/ttlworker/scan.go

```diff
@@ -380,6 +384,10 @@ func (w *ttlScanWorker) handleScanTask(tracer *metrics.PhaseTracer, task *ttlSca
 		result = task.result(nil)
 	}
 
+	if intest.InTest && ctx.Value(TTLScanPostScanHookForTest{}) != nil {
+		ctx.Value(TTLScanPostScanHookForTest{}).(func())()
```

Comment:
Could we call the function like `failpoint.InjectCall`? Then we needn't add a new struct.

Reply:
It's more suggested to use context to inject error when it's possible, right? :thinking:, so that we'll not need to `make failpoint-enable` to run this test.